### PR TITLE
SPEXTRO

### PR DESCRIPTION
**Pull request by wagnerdylan**
### Authors
* Dylan Wagner @VStrideUltimate
* T.J. Tarazevits @venku122
* Matthew Glazer @msglazer

### Abstract

This describes RIT Space Exploration participation in the SDL Payload Challenge at the Spaceport America Cup 2019. A 3U cubesat-like payload featuring a protein spectroscopy experiment will be launched and recovered.

### CODE REVIEW

**Comment by keshavadhyay** [comment]
Nice work. 2 quick points:
1.  Google Docs ' and " problems appear to still be there. This is because GDocs uses some other character. Error manifests in VII-C, 4th line.
2. Is the acknowledgements deliberately mentioning Destler over Munson?

**Comment by wagnerdylan** [comment]
Ok looking good now, added and fixed the bib section.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+{Linden \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+This describes RIT Space Exploration participation in the SDL Payload Challenge at the Spaceport America Cup 2019. A 3U cubesat-like payload featuring a protein spectroscopy experiment will be launched and recovered.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: ~~This describes~~ This project definition document describes

Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: I think it would flow better to say "Participation in the SDL Payload Challenge at the Spaceport America Cup by the RIT Space Exploration Team in 2019".

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} Space Dynamics Laboratory (SDL) payload challenge 2019 is held at the Intercollegiate rocket engineering challenge (IREC) in Las Cruces, New Mexico. The SDL payload challenge is a sub competition at IREC designed to "Encourage participants to create payloads that accomplish a relevant function and provide useful learning opportunities". Throughout the competition, team will be judged on: Scientific or Technical Objective(s), Payload Construction and Overall Professionalism, Readiness / Turnkey Operation and Execution of Objective(s) ~\cite{sdl}. Following this judging criteria, teams will be slotted into first, second and third place with selected others receiving honorable mentions.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: Intercollegiate Rocket Engineering ~~Challenge~~ Competition

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: You might also want to mention that it is held at Spaceport America

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: teams not team

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: Lots of capitalization under the judging criteria. Remove it or turn it into a bulleted list

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+The SDL/IREC team formed within RIT SPEX will work in collaboration with RIT Launch Initiative (LI). LI will design, test and construct a sounding rocket with a target altitude of 30,000ft. The sounding rocket developed by RIT LI will fit a 3U CubeSat form factor payload deployable at apogee, the maximum altitude the rocket achieves.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: Although you have properly introduced each one of these acronyms earlier, they are overused here and will likely confuse the reader. I would rephrase to something like:
"RIT Space Exploration's scientific payload team will once again be working closely with RIT's Launch Initiative (LI) team."

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: The 30k ft number is a goal apogee, not the max. Are you sure that you are still deploying this coming year? Does this payload need to be deployed in order to work? You might not want to commit to this prematurely.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+The goal of this project is to test how high acceleration and zero net acceleration (freefall) affects the folding of proteins as a payload for the SDL Payload Challenge at IREC 2019. To accomplish this, the team will construct a microfluidic spectroscopy experiment that fits within the payload restrictions of the competition. The payload itself must be safe to fly within the competition launch vehicle, provided by RIT Launch Initiative, and should be capable of operating successfully in the desert environment of Las Cruces, as well as safely returning to Earth intact as specified by SDL competition rules.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: You should start by outlining the main goal which is to test how acceleration affects the folding of proteins. Then, break it down into launch and free-fall and explain why this information is desirable.

Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: The flow is off when talking about the restrictions of the competition. It might work better if you turned it into a list.
"This goal will be accomplished constructing a scientific payload that satisfies the requirements outlined by ESRA:
* Thing 1
* Thing 2
...etc."

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+
+\section{Secondary Objective}
+\label{sec:secondary-obj}
+
+This project builds upon the experience and work done for IREC 2018. The  IREC 2018 Technical Report identifies several key areas for improvement that should be pursued. Testing methodology and execution should be emphasized. Several system defects were only discovered during final system integration. Component and subsystem test plans should be formulated and executed well before the deadline to hand-off the payload to the launch team.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: Should source the IREC 2018 technical report in the first sentence

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+
+Ease of use in manufacturing, assembly, and operation should be considered paramount. For most payload types, mass margins are sufficient enough to trade mass saving designs and machining work for quicker production times. Experience from IREC 2018 clearly identifies the need to have physical access to the internals of the payload frequently over the course of the launch campaign. Consideration should be spent on ease of access and speed of final assembly and disassembly. Finally, due to the nature of the IREC competition, payloads can spend extended periods of time loaded in rockets prior to a launch opportunity (> 2 hours). The system should allow for inert stowage in the launch vehicle, until the rocket is on the launch pad.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: ~~Experience~~ The experience

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: In this block, you're talking about things that were learned and need to be changed as a result of what was learned last year. The last sentence isn't really something that wasn't accounted for. It would  be better to say that the goal for runtime now includes the potential for staying in the desert overnight (~24 hrs or more total). With this, power-saving considerations should be made as part of the new system architecture.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+
+Two-way Command and Data Handling (C\&DH) should be included and improvements to transmit/receive throughput should be considered. Signal interrupts and the overhead of task switching between radio modes led to less than expected data throughput. Data received by the ground station was critical in understanding payload performance, since the physical data was lost due to external recovery complications.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: You only use "C\\&DH" in 1 other spot and "\\&" characters should not be included in the acronym. If it stays, at least change it to "CDH"

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: It's not incredibly clear here why the communications were not great and how they need to be improved. You should instead highlight how with only a single radio, transmit and receive modes needed to be toggled and it made things difficult with the different mission states. As a result, we need two radios in the future so that one can perpetually stay in transmit mode while the other stays in receive mode.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+
+Recovery operations are a critical requirement for all IREC and SDL competitors. Due to external factors, the IREC 2018 team was unable to recover the payload, although they believe it landed softly after launch. Known concerns with the payload recovery hardware should be investigated and solutions implemented. Specifically, GPS tracking during the competition was unreliable and the root cause is not known. Also the deployment mechanism for the parachute proved to be unreliable during pre-flight integration. This is a relatively common issue experienced by many teams at IREC 2018. The parachute system should be modified to improve reliability, reduce the chance of accidental deployment, and increase confidence in mission success.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: I would mention why it wasn't recovered since its important to design around that potential in the future. It was also confirmed to have a deployed parachute, landing GPS coordinates (within 100 feet) and was physically spotted by another ground team before it was blown away.

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: Again with the GPS, be specific about how it was unreliable. The coordinates were fine but there wasn't a correct altitude. Mention that a new module will be considered for the next mission.

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: Once again, be *specific* with the issues that were experienced. The parachute deployment was unreliable because of a commercial sensor that had a light sensitivity. In the future, this component (if used) should remain out of the direct sunlight.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+Competing in the Spaceport America Cup provides extensive benefits to SPEX. Firstly, the competition is internationally recognized, with many prestigious engineering schools competing with extensive funding and faculty involvement. Competing at this level shows RIT faculty and staff, potential corporate sponsors and recruiters, and potential members that SPEX is serious about engaging in multidisciplinary engineering projects. A successful payload will require challenging engineering, but also discipline, organization, and creativity; all valuable skills. Spaceport America Cup provides a fantastic learning opportunity for the members who are able to go and many lessons that can be brought back and shared. Hundreds of passionate engineering teams like SPEX are congregated in one place, sharing tips, tricks, advice, and stories of adversity that allow SPEX to learn and evolve to be more effective in the future.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:162]: "engaging in multidisciplinary engineering projects *at at a competitive, international level*"

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:162]: "A successful *mission* will *involve a difficult engineering endeavor that requires high levels of organization, creativity, and technical ability*"

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+
+Also, this project will integrate with other SPEX activities, such as HAB, SPEX-sponsored MSD projects, and other potential test equipment and developed technologies like HABnet. This provides opportunities for members not directly involved with the project to get excited and inspired by the work being done.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:164]: You are using acronyms here that were never introduced. I would suggest just writing the extended version here (for the first time) and introducing the acronym that way. Its also a good idea to use the nomenclature to specify these acronyms at the beginning so the reader can reference them.

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:164]: Not the best to start a sentence off with "Also, ...". You should say something like "This project will integrate multiple sub-projects such as ..."

Comment (dmitchellspace) [reply SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:164]: Please cite these projects!

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 11]
```diff
@@ -0,0 +1,278 @@
...
+
+Historically, SPEX membership has mainly been dominated by physical engineering disciplines. A project of this type holds opportunities for involvement from members with fields of study in bio or chemical engineering. This allows SPEX to cater to a wider range of disciplines further diversifying membership opportunities.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: ~~mainly~~
"physical engineering disciplines *like mechanical and electrical engineering*"
"A project ~~of this type~~ *this diverse*"
~~bio~~ biological
"allows SPEX to ~~cater~~ *be more accessible*"
"wider range of disciplines *that further diversify* membership opportunities"

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 12]
```diff
@@ -0,0 +1,278 @@
...
+Also, this project will integrate with other SPEX activities, such as HAB, SPEX-sponsored MSD projects, and other potential test equipment and developed technologies like HABnet. This provides opportunities for members not directly involved with the project to get excited and inspired by the work being done.
+
+Historically, SPEX membership has mainly been dominated by physical engineering disciplines. A project of this type holds opportunities for involvement from members with fields of study in bio or chemical engineering. This allows SPEX to cater to a wider range of disciplines further diversifying membership opportunities. 
+
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+  
+The payload will iterate and expand on a RIT Multidisciplinary Senior Design Project P16104: Microfluidic Spectroscopy for Proteins within CubeSats ~\cite{msd} conducted during the 2015-2016 school year by SPEX members. The experiment measures the emission spectra generated by proteins after exposure to light. The concept of operations is defined below.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: ~~a~~

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 13]
```diff
@@ -0,0 +1,278 @@
...
+
+\subsection{Concept of Operations - Science}
+\label{subsec:operations}
+A sample of lyophilized protein which contains tryptophan is mixed to reinstate the protein. Once mixing is completed a 280nm LED is turned on to provide an input to the sample. If the sample has folded then the tryptophan will be excited by the 280nm LED and emit light at approximately 350nm. A photodiode is used to measure intensity of the emitted wavelength.
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The project team will produce a qualifying entry into the SDL Payload Challenge 2019.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:196]: What about other deliverables? Code, documentation, schematics, 3-D models, ImagineRIT posters, test results, etc.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 14]
```diff
@@ -0,0 +1,278 @@
...
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+  
+The payload will iterate and expand on a RIT Multidisciplinary Senior Design Project P16104: Microfluidic Spectroscopy for Proteins within CubeSats ~\cite{msd} conducted during the 2015-2016 school year by SPEX members. The experiment measures the emission spectra generated by proteins after exposure to light. The concept of operations is defined below. 
+
+\subsection{Concept of Operations - Science}
+\label{subsec:operations}
+A sample of lyophilized protein which contains tryptophan is mixed to reinstate the protein. Once mixing is completed a 280nm LED is turned on to provide an input to the sample. If the sample has folded then the tryptophan will be excited by the 280nm LED and emit light at approximately 350nm. A photodiode is used to measure intensity of the emitted wavelength.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex]: What is lyophilized protein? 
What is tryptophan? 
What does it mean for a protein to be "reinstated"? 
How is the protein being contained? 
How much of the protein is being contained? 
What is the response rate of the protein? 
How often will this be sampled during the mission? I
s protein folding good or bad? 
What conditions will result in bad data (remember we are in a desert inside a rocket)? 
How are you handling those conditions? 
What is your control/baseline?

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 15]
```diff
@@ -0,0 +1,278 @@
...
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The project team will produce a qualifying entry into the SDL Payload Challenge 2019. 
+
+\subsection{Payload Module}
+\label{subsec:payload}
+The payload module is a maximum 3U (10x10x30cm) module that includes the experiment hardware, telemetry collection and transmission system, and power system for the whole payload.
```
Comment (dmitchellspace) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:200]: a discussion and 3-D model of last year's payload would be helpful here. Then talk about what you are keeping and what you will be getting rid of. How will your model change from the model flown this past year?

**Review by dmitchellspace — COMMENTED** [review]
Only got up to the System Software section. I'll review the rest later.

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 16]
```diff
@@ -0,0 +1,304 @@
...
+\usepackage{siunitx} % enable package for easily formatting units
+% how to use hyperref: http://www2.washjeff.edu/users/rhigginbottom/latex/resources/lecture09.pdf
+\usepackage[T1]{fontenc} % change text encoding to make it more crisp
+\usepackage{etoolbox} % enable conditionals for help text
+\usepackage{booktabs} % make beautiful tables!
+\usepackage{hyperref} % enable package for cross-referencing figures, sections, references etc.
+
+
+% initialize nomenclature package
+\makenomenclature{}
```
Comment (runphilrun) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:33]: delete this command (since you're not using the nomenclature package)

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 17]
```diff
@@ -0,0 +1,304 @@
...
+{Linden \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+This project definition document describes RIT Space Exploration Participation in the SDL Payload Challenge at the Spaceport America Cup by the RIT Space Exploration Team in 2019. A 3U cubesat-like payload featuring a protein spectroscopy experiment will be launched and recovered.
```
Comment (runphilrun) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:100]: 👌

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 18]
```diff
@@ -0,0 +1,304 @@
...
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} Space Dynamics Laboratory (SDL) payload challenge 2019 is held at the Intercollegiate rocket engineering competition (IREC) in Las Cruces, New Mexico. The SDL payload challenge is a sub competition at IREC designed to "Encourage participants to create payloads that accomplish a relevant function and provide useful learning opportunities". Throughout the competition, teams will be judged on: 
+\begin{itemize}
+	\item Scientific or Technical Objective(s)
+	\item Payload Construction and Overall Professionalism 
+	\item Readiness / Turnkey Operation and Execution of Objective(s) ~\cite{sdl}
```
Comment (runphilrun) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:126]: The citation didnt apply the citation here. Make sure you compile with bibtex in the mix. The recipe is `latex bibtex latex latex` https://tex.stackexchange.com/questions/100652/easier-than-latex-bibtex-latex-latex

File: SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex [hunk 19]
```diff
@@ -0,0 +1,304 @@
...
+
+Ease of use in manufacturing, assembly, and operation should be considered paramount. For most payload types, mass margins are sufficient enough to trade mass saving designs and machining work for quicker production times. Experience from IREC 2018 clearly identifies the need to have physical access to the internals of the payload frequently over the course of the launch campaign. Considerations should be made about ease of access and speed of final assembly and disassembly. Finally, due to the nature of the IREC competition, payloads can spend extended periods of time loaded in rockets prior to a launch opportunity (> 2 hours). The system should allow for inert storage in the launch vehicle, until the rocket is on the launch pad.
+
+Two-way Command and Data Handling (C\&DH) should be included and improvements to transmit and receive throughput should be considered. Signal interrupts and the overhead of task switching between radio modes led to less than expected data throughput. Data received by the ground station was critical in understanding payload performance, since the physical data was lost due to external recovery complications.
```
Comment (runphilrun) [inline SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex:150]: The reason to include data transmitted back to ground during flight is great, but why two-way comms (more specifically why does the payload need to also receive)? Do you need to remotely send information or commands to the payload?

Having it on there in order to demonstrate 2way comms is fine but does add some extra complexity. It's not apparent that you need more than 1 way comms (payload-->ground)

**Review by runphilrun — APPROVED** [review]
Dan left some great comments too. I think it's good to merge after taking these comments into consideration
